examples/flutter_gallery/lib/demo/grid_list_demo.dart

```diff
@@ -310,9 +310,6 @@ class GridListDemoState extends State<GridListDemo> {
     });
   }
 
-  // When the ScrollableGrid first appears we want the last row to only be
-  // partially visible, to help the user recognize that the grid is scrollable.
-  // The GridListDemoGridDelegate's tileHeightFactor is used for this.
```

Comment:
We don't appear to have been doing it.  Do you want me to file a bug about adding that functionality?